examples/autopilot_server/autopilot_server.cpp

```diff
@@ -154,7 +161,7 @@ int main(int argc, char** argv)
     // to communicate with the autopilot server plugins.
     mavsdk::Mavsdk mavsdk{Mavsdk::Configuration{Mavsdk::ComponentType::GroundStation}};
 
-    auto result = mavsdk.add_any_connection("udp://:14551");
+    auto result = mavsdk.add_any_connection(argv[1]);
```

Comment:
So what do you use to run the example? I'm a bit confused in terms of connections. And what do you mean when you say "Gazebo simulatoin connection issue"? How are you connecting this to Gazebo?

Reply:
When running PX4 on Gazebo using the command make px4_sitl gazebo_x500, it did not connect because it was using port 14551. Changing it to port 14550 allowed the connection to succeed. `argv[1]  = udp://:14540`

Reply:
But this is not supposed to connect to that case. It is supposed to simulate an autopilot by itself. If you run PX4 Gazebo then you already have an autopilot, and you shouldn't have to use the MAVSDK one. That's why I'm confused.